test/ServerTest.php

```diff
@@ -253,6 +254,16 @@ public function testListenPassesCallableArgumentToCallback()
             ->method('getHeaders')
             ->will($this->returnValue([]));
 
+        $responseBody = new Stream('php://temp');
+        $this->response
+            ->expects($this->any())
+            ->method('getBody')
+            ->willReturn($responseBody);
+        $this->response
+            ->expects($this->any())
+            ->method('withHeader')
+            ->willReturnSelf();
+
```

Comment:
Not really happy with that, mocking the whole world to get an integration test passing isn't really great. Didn't have a lot of time to think of a better solution.